src/modules/generateRandomNumber.js

```diff
@@ -7,7 +7,13 @@
  * @return {number} A random 4-digit number
  */
 function generateRandomNumber() {
-  /* Write your code here */
+  const randomNumber = Math.floor(Math.random() * (10000 - 1) + 1);
```

Comment:
The current logic can generate numbers with fewer than 4 digits and numbers starting with 0. Consider adjusting the range to ensure the generated number is a 4-digit number that does not start with 0.